client/mainmenu/CHighScoreScreen.h

```diff
@@ -70,16 +71,19 @@ class CHighScoreInput : public CWindowObject
 
 class CHighScoreInputScreen : public CWindowObject
 {
-	std::vector<std::shared_ptr<CMultiLineLabel>> texts;
+	std::vector<std::shared_ptr<CIntObject>> texts;
```

Comment:
Why texts can't be represented with a label and require CIntObject? Some texts are not actually texts?